app/views/fe/admin/panels/_prop_sheet.html.erb

```diff
@@ -10,8 +10,11 @@
     <p><label for="" class="desc">Label <span class="required">required</span></label>
     <%= f.text_field :label, :maxlength => 50 %></p>
 
+    <p><label for="" class="desc">Languages (in addition to english)</label>
+    <%= f.select :languages, Fe::LANGUAGES.collect {|k, v| [v, k]}, {}, { size: 3, multiple: true }  %></p>
+
     <p class="buttons">
-    <%= submit_tag"Update &raquo;".html_safe %></p>
+    <%= submit_tag"Update &raquo;".html_safe, data: { disable_with: "Please wait..." } %></p>
```

Comment:
Might as well put a space after submit_tag? It looks a bit odd (I know it was like that before..)